packages/babel-plugin-transform-es2015-modules-umd/src/index.js

```diff
@@ -39,7 +39,7 @@ export default function ({ types: t }) {
 
     visitor: {
       Program: {
-        exit(path) {
+        exit(path, plugin) {
```

Comment:
Can you rename `plugin` `state`, just to be consistent with how this is named elsewhere